### ISSUE TITLE
Learning rate scheduler

### ISSUE THREAD

**Issue by blythed** [blythed/aleph#13]
This comes with pytorch, we need to add it to the trainer.

**Comment by blythed** [comment]
Solved
